src/main/views/court-details/not-in-person-court.njk

```diff
@@ -62,66 +62,57 @@
           <div class="govuk-grid-row .contacts" id="contacts">
             <h4 class="govuk-heading-m">
-              {% if results.enquiries.fax.number %}
+              {% if results.enquiries.fax.length %}
                 {{ telephoneAndFaxHeading }}
               {% else %}
                 {{ telephoneHeading }}
               {% endif %}
             </h4>
-            {% for telephone in results.enquiries.phone %}
-              <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
-                  <p class="govuk-body-m">
-                    {{ telephone.description}}
-                  </p>
-                </div>
-                <div class="govuk-grid-column-two-thirds">
-                  <div class="govuk-body-m">
-                    <a class="govuk-link" href="tel:{{ telephone.number }}">{{ telephone.number }}</a>
-                    {% if telephone.explanation %}
-                      </br>
-                      <p class="govuk-body-m">({{ telephone.explanation }})</p>
-                    {% endif %}
-                  </div>
-                </div>
-              </div>
-            {% endfor %}
-            {% for telephone in results.enquiries.welshPhone %}
-              <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
-                  <p class="govuk-body-m">
+
+            <dl class="govuk-summary-list">
+              {% for telephone in results.enquiries.phone %}
+                <div class="govuk-summary-list__row">
+                  <dd class="govuk-summary-list__key">
                     {{ telephone.description}}
-                  </p>
-                </div>
-                <div class="govuk-grid-column-two-thirds">
-                  <div class="govuk-body-m">
-                    <a class="govuk-link" href="tel:{{ telephone.number }}">{{ telephone.number }}</a>
-                    {% if telephone.explanation %}
-                      </br>
-                      <p class="govuk-body-m">({{ telephone.explanation }})</p>
-                    {% endif %}
-                  </div>
+                  </dd>
+                  <dd class="govuk-summary-list__value">
+                   <a class="govuk-link" href="tel:{{ telephone.number }}">{{ telephone.number }}</a>
+                   {% if telephone.explanation %}
+                     <p class="govuk-body-m">({{ telephone.explanation }})</p>
+                   {% endif %}
+                  </dd>
                 </div>
-              </div>
-            {% endfor %}
-            {% for fax in results.enquiries.fax %}
-              <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
-                  <p class="govuk-body-m">
-                    {{ fax.description}}
-                  </p>
+              {% endfor %}
+
+              {% for telephone in results.enquiries.welshPhone %}
```

Comment:
There are 3 sub-sections for the telephone number section, i.e. the enquiries phone, the welsh phone and fax. For all not-in-person courts, I can't see any phone number which has the word 'Welsh' in the phone description. There are some with the word "Welsh" in the explanation field but this is not used to filter the phone type in CourtDetailController.ts. 

I think there is only one not-in-person court with Fax number (CCMCC) and this is already sorted last without the need of filtering and grouping into sub-sections. The sorting should ideally be done by the sort column in the database and not hard-coded logic like this. I got a feeling without this bespoke sorting logic here, it is already sorted in the right order. I wonder we should try removing the filtering in CourtDetailController.ts and just use results.contacts for displaying the phone numbers.

Reply:
This strange logic is existing behaviour. What you have said makes sense and something I had thought but I've presumed that this has been done for a reason.

Reply:
I think we should keep bespoke logic in the front-end to minimum if that can be done in the backend. In this case, the sorting can be done in the database (either directly in database or through sorting through admin postal), we can keep the logic here simpler if we can.

Reply:
Had a chat with @Thomas-Geraghty and he's of the view we should sort the phone numbers now. I'll make the necessary changes this afternoon.

Reply:
I think it is already sorted correctly if we just use results.contacts in the nunjunks file...but u will need to confirm after the change